commerce-notification-test/src/testIntegration/java/com/liferay/commerce/notification/test/CommerceOrderStatusNotificationTest.java

```diff
@@ -0,0 +1,257 @@
+
+		Assert.assertEquals(2, commerceNotificationQueueEntriesCount);
+
+		_commerceOrderLocalService.updateOrderStatus(
+			_commerceOrder.getCommerceOrderId(),
+			CommerceOrderConstants.ORDER_STATUS_SHIPPED);
+
+		// Notifications are asynchronous, give time to send
+
+		Thread.sleep(1000);
+
+		commerceNotificationQueueEntriesCount =
+			_commerceNotificationQueueEntryLocalService.
+				getCommerceNotificationQueueEntriesCount(_group.getGroupId());
+
+		Assert.assertEquals(3, commerceNotificationQueueEntriesCount);
+
+		_commerceOrderLocalService.updateOrderStatus(
+			_commerceOrder.getCommerceOrderId(),
+			CommerceOrderConstants.ORDER_STATUS_COMPLETED);
+
+		// Notifications are asynchronous, give time to send
+
+		Thread.sleep(1000);
+
+		commerceNotificationQueueEntriesCount =
+			_commerceNotificationQueueEntryLocalService.
+				getCommerceNotificationQueueEntriesCount(_group.getGroupId());
+
+		Assert.assertEquals(4, commerceNotificationQueueEntriesCount);
+	}
+
+	private void _addNotificationTemplates() throws PortalException {
+		_notificationTemplateList = new ArrayList<>(4);
+
+		CommerceNotificationTemplate commerceNotificationTemplate =
+			CommerceNotificationTestUtil.addNotificationTemplate(
+				"[%ORDER_CREATOR%]",
+				CommerceOrderConstants.ORDER_NOTIFICATION_PLACED,
+				_serviceContext);
+
+		_notificationTemplateList.add(commerceNotificationTemplate);
+
+		commerceNotificationTemplate =
+			CommerceNotificationTestUtil.addNotificationTemplate(
+				"[%ORDER_CREATOR%]",
+				CommerceOrderConstants.ORDER_NOTIFICATION_AWAITING_SHIPMENT,
+				_serviceContext);
+
+		_notificationTemplateList.add(commerceNotificationTemplate);
+
+		commerceNotificationTemplate =
+			CommerceNotificationTestUtil.addNotificationTemplate(
+				"[%ORDER_CREATOR%]",
+				CommerceOrderConstants.ORDER_NOTIFICATION_SHIPPED,
+				_serviceContext);
+
+		_notificationTemplateList.add(commerceNotificationTemplate);
+
+		commerceNotificationTemplate =
+			CommerceNotificationTestUtil.addNotificationTemplate(
+				"[%ORDER_CREATOR%]",
+				CommerceOrderConstants.ORDER_NOTIFICATION_COMPLETED,
+				_serviceContext);
+
+		_notificationTemplateList.add(commerceNotificationTemplate);
+	}
+
+	private CommerceAccount _commerceAccount;
+	private CommerceAccountGroup _commerceAccountGroup;
+
+	@Inject
+	private CommerceChannelLocalService _commerceChannelLocalService;
+
+	private CommerceCurrency _commerceCurrency;
+
+	@Inject
+	private CommerceNotificationQueueEntryLocalService
+		_commerceNotificationQueueEntryLocalService;
+
+	@Inject
+	private CommerceNotificationTemplateLocalService
+		_commerceNotificationTemplateLocalService;
+
+	@DeleteAfterTestRun
+	private CommerceOrder _commerceOrder;
+
+	@Inject
+	private CommerceOrderLocalService _commerceOrderLocalService;
+
+	@DeleteAfterTestRun
+	private Company _company;
+
+	private Group _group;
+	private List<CommerceNotificationTemplate> _notificationTemplateList;
```

Comment:
`List<CommerceNotificationTemplate> _notificationTemplateList` should be `List<CommerceNotificationTemplate> _commerceNotificationTemplates`